components/eamxx/src/physics/spa/tests/spa_main_test.cpp

```diff
@@ -192,9 +192,9 @@ TEST_CASE("spa_main")
   std::cout << "  -> time interp, t_beg<t<t_end\n";
 
   RPDF pdf(0.001,0.999);
-  auto dt = int ( (t_end-t_beg) * pdf(engine) );
+  auto dt = (t_end-t_beg) * pdf(engine);
 
-  auto t_mid = t_beg + dt;
+  auto t_mid = t_beg + static_cast<int>(dt);
```

Comment:
`round` here, too.

Reply:
But do you actually need this to be an `int`? Below I see the time class can handle `double`.

Reply:
This is for a unit test, so I wasn't really concerned about the +/- one. But I can change it to the nearest one.